oam/install_scripts/mcs-stop-controllernode.sh.in

```diff
@@ -4,12 +4,12 @@
 systemctl stop mcs-workernode --ignore-dependencies
 timeout=$(mcsGetConfig SystemConfig DBRMTimeout)
 
-while [ -n $(pgrep -x controllernode) ] && [ $timeout -gt 0 ]
+while [ -n "$(pgrep -x controllernode)" ] && [ $timeout -gt 0 ]
```

Comment:
Be safe here. If pgrep -x controllernode returns nothing it will not treat it as an empty string